src/garage/torch/algos/sac.py

```diff
@@ -367,6 +389,27 @@ def optimize_policy(self, itr, samples_data):
 
         return policy_loss, qf1_loss, qf2_loss
 
+    def _evaluate_policy(self, epoch):
+        """Evaluate the performance of the policy via deterministic rollouts.
+
+            Statistics such as (average) discounted return and success rate are
+            recorded.
+
+        Args:
+            epoch(int): The current training epoch.
+
+        Returns:
+            float: The average return across self._num_evaluation_trajs
+                trajectories
+
+        """
+        last_return = log_performance(
+            epoch,
+            self._obtain_evaluation_samples(
```

Comment:
this indentation is tortured. just retreive the samples in a different block -- it costs nothing.